frontend/packages/integration-tests-cypress/views/list-page.ts

```diff
@@ -148,12 +154,10 @@ export const listPage = {
           cy.byTestID('popover-status-button').click();
         });
     },
-    shouldExist: (resourceName: string) => {
-      cy.get(`[data-test="data-view-cell-${resourceName}-name"]`)
-        .contains(resourceName)
-        .should('exist');
+    shouldExist: (resourceName: string, cellName: string = 'name') => {
+      cy.get(`[data-test="data-view-cell-${resourceName}-${cellName}"]`).should('exist');
     },
-    shouldNotExist: (resourceName: string) => {
+    shouldNotExistWithName: (resourceName: string) => {
```

Comment:
Update this the same way you did `shouldExist` and then you don't need to rename.